verticaltabs.js

```diff
@@ -984,35 +982,16 @@ VerticalTabs.prototype = {
     case 'TabClose':
       this.onTabClose(aEvent);
       return;
-    case 'TabPinned':
-      this.onTabPinned(aEvent);
-      return;
-    case 'TabUnpinned':
-      this.onTabUnpinned(aEvent);
-      return;
-    case 'mouseup':
```

Comment:
Should we also remove the `onMouseUp` function, or does that get used somewhere else?